pyOCD/tools/pyocd.py

```diff
@@ -33,17 +36,135 @@
 except ImportError:
     isCapstoneAvailable = False
 
-# Command action constants.
-ACTION_LIST = 1
-ACTION_ERASE = 2
-ACTION_UNLOCK = 3
-ACTION_INFO = 4
-ACTION_PROGRAM = 5
-ACTION_RESET = 6
-ACTION_READ = 7
-ACTION_WRITE = 8
-ACTION_GO = 9
-
+LEVELS = {
+        'debug':logging.DEBUG,
+        'info':logging.INFO,
+        'warning':logging.WARNING,
+        'error':logging.ERROR,
+        'critical':logging.CRITICAL
+        }
+
+CORE_STATUS_DESC = {
+        pyOCD.target.cortex_m.TARGET_HALTED : "Halted",
+        pyOCD.target.cortex_m.TARGET_RUNNING : "Running"
+        }
+
+## Default SWD clock in kHz.
+DEFAULT_CLOCK_FREQ_KHZ = 1000
+
+## Command info and help.
+COMMAND_INFO = {
+        'list' : {
```

Comment:
Maybe it could be popped from help at the same time it is popped from command_list